src/syntax_util.ml

```diff
@@ -83,6 +83,10 @@ let reason_to_ml_swapping_alist = [
   "\\===", "===";
   "!=", "<>";
   "!==", "!=";
+  "switch", "match";
+  "match", "switch";
```

Comment:
I thought this list already implies the bidirectional. For example, the first entry ` "===",  "=="` means:

> In Reason, when you write `===`, it means the ML AST identifier `==`. Therefore, the ML identifier `==` is printed as `===` when printing Reason.

So, I *think* that you only need ` "match", "switch"`, which means:

> In Reason, when you write `match`, it means the ML AST identifier `switch`. Therefore, the ML identifier `switch` is printed as `match` when printing Reason.

There is *one* AST we are describing here - the ML AST.  In that ML AST, it is (apparently) impossible to express the identifier named `match`:

```ocaml
utop # let match = "asdf";;
Error: Syntax error
```

So therefore, it shouldn't be necessary to include `"switch", "match";`. Do I understand correctly?